fl4health/server/scaffold_server.py

```diff
@@ -22,37 +22,39 @@ def __init__(
         strategy: Scaffold,
         checkpointer: Optional[TorchCheckpointer] = None,
         reporters: Sequence[BaseReporter] | None = None,
-        warm_start: bool = False,  # Whether or not to initialize control variates of each client as local gradient
+        warm_start: bool = False,
+        accept_failures: bool = True,
     ) -> None:
         """
-        Custom FL Server for scaffold algorithm to handle warm initialization of control variates
-        as specified in https://arxiv.org/abs/1910.06378.
+        Custom FL Server for scaffold algorithm to handle warm initialization of control variates as specified in
+        https://arxiv.org/abs/1910.06378.
 
         Args:
             client_manager (ClientManager): Determines the mechanism by which clients are sampled by the server, if
                 they are to be sampled at all.
             strategy (Scaffold): The aggregation strategy to be used by the server to handle client updates and
                 other information potentially sent by the participating clients. This strategy must be of SCAFFOLD
                 type.
-            checkpointer (Optional[TorchCheckpointer], optional): To be provided if the
-                server should perform server side checkpointing based on some criteria.
-                If none, then no server-side checkpointing is performed. Defaults to
-                None.
-            reporters (Sequence[BaseReporter], optional): A sequence of FL4Health
-                reporters which the server should send data to before and after each
-                round.
-            warm_start (bool, optional): Whether or not to initialize control variates
-                of each client as local gradients. The clients will perform a training
-                pass (without updating the weights) in order to provide a "warm"
-                estimate of the SCAFFOLD control variates. If false, variates are
-                initialized to 0. Defaults to False.
+            checkpointer (Optional[TorchCheckpointer], optional): To be provided if the server should perform server
+                side checkpointing based on some criteria. If none, then no server-side checkpointing is performed.
+                Defaults to None.
```

Comment:
lol sorry the formatting to a shorter line length was definitely my fault

Reply:
No worries. Just moving it back where I see it 🙂